themes/bootstrap5/templates/Helpers/pagination.phtml

```diff
@@ -1,24 +1,46 @@
 <?php if ($this->pageCount): ?>
   <?php
+    $results = $this->params['results'] ?? null;
     $showFirstLast = $this->pageCount > 3;
+    $navAttrs = $this->htmlAttributes(['aria-label' => $this->translate('pagination_label')]);
+    if ($results !== null) {
+      $showFirstLast = $showFirstLast && $results->getOptions()->firstLastNavigationSupported();
+      if ($results->getOptions()->loadResultsWithJsEnabled()) {
+        $navAttrs->add('class', 'js-pagination');
+      }
+    }
+    if ($this->top) {
+      $navAttrs->add('class', 'js-pagination__top');
+    }
   ?>
-  <nav aria-label="<?=$this->transEscAttr('pagination_label')?>">
-    <ul class="pagination">
-      <?php if ($this->previous): ?>
-        <?php if ($showFirstLast && $this->first): ?>
+  <nav<?=$navAttrs?>>
+    <?php
+      $ulAttrs = $this->htmlAttributes(['class' => 'pagination']);
+      if ($this->top) {
+        $ulAttrs->add('class', 'pagination-top');
+      }
+      if ($className = $this->options['className'] ?? '') {
+        $ulAttrs->add('class', $className);
+      }
+    ?>
+    <ul<?=$ulAttrs?>>
+      <?php if (isset($this->previous)): ?>
```

Comment:
Is there a conscious reason to add `isset` to the checks here and in the next line? This removes our ability to set these values to false, since that is set even though false-y. Not sure if that's a problem, but just pointing out the increased strictness.

Reply:
(see also next/last below for more instances of the same thing).

Reply:
I thought that this might be a problem if `previous` is 0. But since the pages should start with 1 that probably can never happen.

It was also done like this here https://github.com/vufind-org/vufind/blob/6495f63ce7f0923c63d0e692b7ba462945ff3782/themes/bootstrap5/templates/search/pagination.phtml#L23 where I copied a lot of the changes from. 

I am certainly open to remove `isset` here.

Reply:
Sorry, I had forgotten that `next` and `previous` were page numbers and not simply boolean indicators. I think it's fine to leave this as-is.